modules/twinklewarn.js

```diff
@@ -1154,6 +1154,12 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 	// Limit the max height of select dropdown to prevent dialog box from becoming scrollable
 	$('.chosen-results').css({'overflow': 'auto', 'max-height': '180px'});
 
+	// Remove padding
+	mw.util.addCSS('.chosen-drop .chosen-results li { padding-top: 0px; padding-bottom: 0px; }');
```

Comment:
I think let's just keep it simple and do 1px across the board, i.e. remove the fancy bit for search results
```suggestion
	mw.util.addCSS('.chosen-drop .chosen-results li { padding-top: 1px; padding-bottom: 1px; }');
```

Reply:
why? The keyup handler may look a bit expensive but it doesn't cause any delays or anything. 0px for the default case is closest to the native select menu and packs in an extra template in the view without scrolling down.

Reply:
I meant it less as favoring 1px over 0px and more as it's unnecessary to do the dance to change the menu on the fly.  1 or 0 is fine with me (I probably favor 1px slightly) but I'd sooner not change the menu.  Going 0 everywhere is fine IMO.